crucible-go/crucible-go.cabal

```diff
@@ -17,27 +17,10 @@ executable crux-go
 
   build-depends:
     base >= 4 && < 5,
-    aig,
-    ansi-wl-pprint,
-    array,
-    containers,
     crucible,
-    directory,
-    filepath,
-    haskeline >= 0.7,
-    lens,
-    mtl >= 2.1,
     parameterized-utils >= 1.0 && < 2.2,
-    pretty >= 1.1,
-    split >= 0.2,
-    text,
-    transformers >= 0.3,
-    transformers-compat,
-    vector >= 0.7,
     golang,
-    what4 >= 0.4,
```

Comment:
Removing these from this section drops some version constraints that should be moved to the area below where they still apply (at least `vector` and `what4`, but probably should check all others.